src/System.Net.Primitives/src/System/Net/Cookie.cs

```diff
@@ -43,6 +42,8 @@ public sealed class Cookie
         internal const int MaxSupportedVersion = 1;
         internal const string MaxSupportedVersionString = "1";
 
+        internal const string NameAttributeName = "Name";
+        internal const string ValueAttributeName = "Value";
```

Comment:
For now, please leave this PR as a tests only cleanup. Please remove product code changes from the PR.